spec/directives/for/basic.hrx

```diff
@@ -0,0 +1,145 @@
+<===> inclusive_forward/input.scss
```

Comment:
It's a good idea to [avoid placeholder path names](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#do-use-descriptive-paths) like "basic", and try to find a more descriptive way to divide up the tests instead. In this case, you can just put everything under a single `for.hrx` file since there aren't that many specs yet.